addons/explosives/functions/fnc_dialPhone.sqf

```diff
@@ -28,22 +26,34 @@ private _arr = [];
 for "_i" from 1 to _ran do {
     _arr = _arr + ['.','..','...',''];
 };
+
+private _explosive = [_code] call FUNC(getSpeedDialExplosive);
+
+private _explosiveType = typeOf (_explosive select 0);
+private _ringtone = GVAR(iedRingtonesHash) getOrDefault [_explosiveType,getArray (configFile >> "CfgAmmo" >> _explosiveType >> QGVAR(ringtones)),true];
+//support random ringtones
```

Comment:
maybe a fallback for ammos that don't have the config?
```suggestion
if (_ringtone isEqualTo []) then { _ringtone = [[QPATHTOF(Data\Audio\Cellphone_Ring.wss), 0.75]] };
//support random ringtones
```

Reply:
yeah, that is good.